huxley/www/static/js/huxley/components/RegistrationView.js

```diff
@@ -89,6 +89,14 @@ var RegistrationView = React.createClass({
     };
   },
 
+  componentWillMount: function() {
```

Comment:
Once the registration handler is set to either open, closed, or waitlist, we won't need this function any more.